docs/tutorials/01_introductory/viz_solar_system.py

```diff
@@ -22,14 +22,20 @@
 ##############################################################################
 # Load in a texture for each of the actors. To do this, we will create
 # a function called ``init_planet``, which will initialize each planet actor
-# given its corresponding filename and actor name. It will also add each
-# actor to the scene that has already been created.
-
-planet_filenames = ["8k_mercury.jpg", "8k_venus_surface.jpg",
-                    "1_earth_8k.jpg", "8k_mars.jpg", "jupiter.jpg",
-                    "8k_saturn.jpg", "8k_saturn_ring_alpha.png",
-                    "2k_uranus.jpg", "2k_neptune.jpg",
-                    "8k_sun.jpg"]
+# given its corresponding filename and actor name.It will also set the
+# position of the planet and rescale it. It will also add each actor to
+# the scene that has already been created.
+
+planet_filenames = {"8k_mercury.jpg": [7, (0.4, 0.4, 0.4)],
```

Comment:
Now that you're changing the array's content, I suggest renaming it, re-organizing it, and re-writing the description. Something like this could do:

`# Define information relevant for each planet actor including its texture name, relative position, and scale...`
`planets_data = [{'filename': '8k_mercury.jpg', 'position': 7, 'scale': (.4, .4, .4)}, 
{'filename': '8k_venus_surface.jpg', 'position': 9, 'scale': (.6, .6, .6)}, ...]`

`# To take advantage of the previously defined data structure we are going to create an auxiliary function that will load and apply the respective texture, set its respective properties (relative position and scale), and add the actor to a previously created scene.`
`def init_planet(planet_data):`

This will make everything more explicit.